sycl/test-e2e/bindless_images/vulkan_interop/mipmaps.cpp

```diff
@@ -147,6 +148,8 @@ bool run_sycl(sycl::range<NDims> globalSize, sycl::range<NDims> localSize,
     exit(-1);
   }
 
+  using bindless_helpers::equal_vec;
+
```

Comment:
This still isn't consistent, as the `init_vector` has to specify the namespace explicitly and the `equal_vec` doesn't.. Can we just remove all `using bindless_helpers::*` and explicitly state which namespace functions are being called from?

Reply:
>This still isn't consistent, as the init_vector has to specify the namespace explicitly and the equal_vec doesn't.. 

it is consistent across interop tests since that I use `using bindless_helpers::` for functions that will be called inside if conditions, e.g `if'(equal_vec(..))`. given that's a matter of preference, I will remove `using ...` from all the interop test.

Reply:
done.